github-electron/github-electron.d.ts

```diff
@@ -1464,6 +1464,24 @@ declare module GitHubElectron {
 		sendToHost(channel: string, ...args: any[]): void;
 	}
 
+	class IPCMain implements NodeJS.EventEmitter {
+		addListener(event: string, listener: Function): IPCMain;
+		once(event: string, listener: Function): IPCMain;
+		removeListener(event: string, listener: Function): IPCMain;
+		removeAllListeners(event?: string): IPCMain;
+		setMaxListeners(n: number): IPCMain;
+		getMaxListeners(): number;
+		listeners(event: string): Function[];
+		emit(event: string, ...args: any[]): boolean;
+		listenerCount(type: string): number;
+		on(event: string, listener: (event: IPCMainEvent, ...args: any[]) => any): IPCMain;
+	}
+
+	interface IPCMainEvent {
+		returnValue?: any;
```

Comment:
https://github.com/atom/electron/blob/v0.36.2/atom/browser/api/lib/web-contents.coffee#L65-L71
`event.returnValue` is not a normal member(it has only setter).
Assignment to `event.returnValue` has no effect on asynchronous events. 
But `sender` Member is always available. I'll fix it.